opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectAskAI.tsx

```diff
@@ -210,7 +211,21 @@ const StixCoreObjectAskAI: FunctionComponent<StixCoreObjectAskAiProps> = ({ inst
         });
         break;
       default:
-        // do nothing
+      // do nothing
+    }
+  };
+  const handleAskAi = () => {
+    // check paragraphs value is correct
+    if (action === 'container-report' || action === 'summarize-files') {
+      if (Number.isNaN(paragraphs)) {
+        MESSAGING$.notifyError('Number of paragraphs should be a number');
```

Comment:
Was it possible to add an error with Yup in the field directly instead of launching a MESSAGING error ?

Reply:
I wanted to but it requires a formik...